src/style.css

```
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.image {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.input {
+  outline: none;
+  font-family: inherit;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.form--small {
+  position: relative;
+}
+
+.input--small {
+  font-weight: 300;
+  font-family: inherit;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```

```suggestion
  font-weight: 300;
  height: 42px;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```
recheck repeated styles everywhere, DRY